src/parser/shared/modules/StatTracker.js

```diff
@@ -684,9 +724,24 @@ class StatTracker extends Analyzer {
       debug && this._debugPrintStats(this._currentStats);
       this._triggerChangeStats(event, before, delta, after);
     }
+    if (statMult) {
+      // ignore prepull buff application, as they're already accounted for in combatantinfo
```

Comment:
oh, you addressed that here. how does this interact with dying and getting AI/shout after res?

Reply:
Dying clears all buffs, meaning this function gets called with `event.newStacks = 0` and 
`event.oldStacks = 1`. ` this.removeStatMultiplier(statMult, true)` will then remove the multiplier as well as the stats provided by that multiplier, and if someone re-applies the buff after reviving, it will be added to the current stats again.